tests/organizations/test_api_update_organization.py

```diff
@@ -83,31 +80,77 @@ def setUp(self, mock_login, mock_get_user):
             "status": "Draft",
         }
 
-        
     def test_api_dao_create_organization_successfully(self):
         test_user_extension = UserExtensionModel(
             user_id=self.test_user1_data.id,
             timezone="AUSTRALIA_MELBOURNE"
         )
         test_user_extension.is_organization_rep = True
         test_user_extension.save_to_db()
-
         response = self.client.put(
             "/organization",
             headers={"Authorization": AUTH_COOKIE["Authorization"].value},
             data=json.dumps(
-                    dict(self.correct_payload_organization)
-                ),
+                dict(self.correct_payload_organization)
+            ),
             follow_redirects=True,
             content_type="application/json",
         )
         self.assertEqual(HTTPStatus.CREATED, response.status_code)
         self.assertEqual(messages.ORGANIZATION_SUCCESSFULLY_CREATED, json.loads(response.data))
 
+    def test_api_dao_create_organization_when_name_not_unique(self):
```

Comment:
@NenadPantelic perhaps rename this to "...._when_name_already_exist" to avoid ambiguity coz organization name is unique.